packages/gatsby-image/src/index.js

```diff
@@ -202,7 +202,12 @@ const noscriptImg = props => {
 // Earlier versions of gatsby-image during the 2.x cycle did not wrap
 // the `Img` component in a `picture` element. This maintains compatibility
 // until a breaking change can be introduced in the next major release
-const Placeholder = ({ src, spreadProps, imageVariants, generateSources }) => {
+const Placeholder = ({
+  src,
+  imageVariants,
+  generateSources,
+  ...spreadProps
```

Comment:
If you're going to do this, might as well stick to the usual conventional name of `...rest`? 

`spreadProps` was a prop name to communicate the intent to spread the props to a child. Which I believe is less ambiguous to the reader vs collecting any additional props(some which can be implicit like the children prop(not a concern here since this is all handled internally).

Does the use of two additional spread syntax (one here, and another per instance where the prop name is now removed) provide value in some way? Your commit message didn't really indicate why this change was made, I'm curious.

Reply:
oh I just feel this is better XD I didn't see much point of not using {...placeholderImageProps} but i'll revert as i'm probably the only one thinking that.